test/ceil.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Maybe rename this file to `test/llvm.ceil.ll` (to make it clearer this is testing an llvm intrinsic)?  Same for the other test.

Reply:
This has already been checked into xmain branch. Name change might cause a bit of headache during pull down later on.

Reply:
> This has already been checked into xmain branch. Name change might cause a bit of headache during pull down later on.

I don't know what the "xmain branch" is, but that is not relevant to this project.  For this project, it would be good to keep some structure in the naming of test files, which is why I made this suggestion.

Reply:
I agree. I will change this.

Reply:
Done